scripts/update-go-pbs.sh

```diff
@@ -13,13 +13,14 @@ while IFS= read -d $'\0' -r file ; do
 done < <(find -L $(bazel info bazel-bin)/proto -type f -name "*pb.go" -print0)
  
 arraylength=${#file_list[@]}
-searchstring="prysm/"
+searchstring="prysmaticlabs/prysm/"
 
 # Copy pb.go files from bazel-bin to original folder where .proto is.
 
 for (( i=0; i<${arraylength}; i++ ));
 do
   destination=${file_list[i]#*$searchstring}
+  chmod 755 "$destination"
```

Comment:
any reason to change to 755 instead of 644 ?

Reply:
i had 755 and 555 in those directories . i just changed it to the one that worked

Reply:
i guess 644 would suffice. thanks